active_publisher.gemspec

```diff
@@ -26,6 +26,8 @@ Gem::Specification.new do |spec|
     spec.add_dependency 'bunny', '~> 2.1'
   end
 
+  spec.add_dependency 'multi_op_queue'
```

Comment:
maybe we should add a `> 0.1.2` requirement on this dependency since that is the version that fixed the problem for earlier versions of ruby 2?